docs/DEBUG_COMMANDS.md

```diff
@@ -0,0 +1,57 @@
+# Available debug commands
+
+Organic Maps exposes debug commands to help you control the native components (engine, editor, navigation, ...). **These commands are not intended for regular users** and are only used for debug purposes. Please only use these commands if you are working on Organic Maps.
+
+Each command is entered in the search input (Android and iOS) and are activated as soon as the full search keyword is entered. Unless specified, the effects triggered are discarded after a restart.
+
+For more information, please see the source code at [`Framework::ParseSearchQueryCommand`](map/framework.cpp).
+
+## Drape
+
+### Themes
+
+- `?dark` or `mapstyle:dark`: Enable night mode for the map view only. You may need to change the zoom level to fully apply the new mode.
+- `?light` or `mapstyle:light`: Enable light mode for the map view only. You may need to change the zoom level to fully apply the new mode.
+- `?vdark` or `mapstyle:vdark`: Same as `?dark`, but using the vehicle variant.
+- `?vlight` or `mapstyle:vlight`: Same as `?light`, but using the vehicle variant.
+
+### Post processing
+
+- `?aa` or `effect:antialiasing`: Enable antialiasing.
+- `?no-aa` or `effect:no-antialiasing`: Disable antialiasing.
+
+### Map layers
+
+- `?scheme`: Enable the subway layer.
+- `?no-scheme`: Disable the subway layer.
+- `?isolines`: Enable the isolines layer.
+- `?no-isolines`: Disable the isolines layer.
+
+### Information
+
+- `?debug-info`: Show renderer version, zoom scale and FPS counter in the top left corner of the map.
+- `?debug-info-always`: Same as `?debug-info`, but persists across restarts.
+- `?debug-rect`: TODO
+- `?no-debug-rect`: TODO.
+
+### Engine
+
+All the following commands require a restart to be applied.
+
+- `?gl`: Enables the OpenGL renderer. Platforms: All.
+- `?vulkan`: Enables the Vulkan renderer. Platforms: Android.
+- `?metal`: Enables the Metal renderer. Platforms: TODO.
```

Comment:
```suggestion
- `?metal`: Enables the Metal renderer. Platforms: iOS and macOS.
```

Reply:
I don't know the Apple ecosystem, do all iphones/Mac support metal?

Reply:
https://en.wikipedia.org/wiki/Metal_(API)